client/src/hooks/AuthContext.tsx

```diff
@@ -139,6 +139,8 @@ const AuthContextProvider = ({
     error,
     navigate,
     setUserContext,
+    doSetError,
```

Comment:
similar to above, weird errors could be introduced here

Reply:
Adding doSetError here is causing a loop, making the call 
http://localhost:3090/api/auth/refresh over and over again. Whether or not the ones above (authConfig etc.) are there don't have any effect on it.
doSetError uses useTimeout() which doesn't have a dependency list.